### DISCUSSION TITLE
Add a command to convert .torrent to magnet link

### DISCUSSION THREAD

**Discussion by mak190045** [AnimMouse/torrent-webseed-creator#11] [Ideas]
Hi,
Can a change be introduced in the file to Add a command to convert .torrent into magnet link. Conversion is widely available on the internet.

**Comment by AnimMouse** [comment]
I think it would be useless if you use magnet link on this, as for magnet links to work, there should be a seeder in order to get the metadata required in downloading the torrent (the hashes of all pieces in the torrent).
As this torrent relies only on the webseed with no seeders initially, you will get stuck downloading the metadata as there is no one is seeding the metadata.

If you somewhat make enough seeders in the torrent you created, you can easily make a magnet link using the infohash if you use torrenttools, torf-cli, or dottorrent-cli.

[Example](https://github.com/AnimMouse/torrent-webseed-creator/runs/2483001710?check_suite_focus=true)
In the "Create torrent using torrenttools," you can see the infohash.
Infohash: 5c84c433313413de6185544cce8ca81f582e1557
Just append magnet:?xt=urn:btih: before the infohash
magnet:?xt=urn:btih:5c84c433313413de6185544cce8ca81f582e1557
You just made a magnet link.
Try it, surely it will not work as no one is seeding it, it will only work if there is 1 seeder on it.
